library-best-practices/JLBP-13.md

```diff
@@ -0,0 +1,25 @@
+[JLBP-13] Quickly remove references to deprecated features in dependencies
+--------------------------------------------------------------------------
+
+- This best practice refers to code that *uses* deprecated functionality from
+  another library, not to the deprecated functionality itself.
+  - Consequently, this best practice does not apply to libraries that have no
```

Comment:
All libraries depend on the JDK and it does have deprecated features. 20 years in some of those are finally being removed. I suggest removing the first clause.

Reply:
Oh of course, I overlooked that. I'll change it.